src/test/counter.test.js

```diff
@@ -0,0 +1,39 @@
+import DOMManipulator from '../modules/dom.js';
+
+const dom = new DOMManipulator();
+
+describe('Test show and comment counters', () => {
+  test('Test show counter', async () => {
+    const shows = [
+      'Under the Dome',
+      'Person fo interest',
+      'Bitten',
+      'The Strain',
+    ];
+    expect(await dom.showCounter(shows)).toBe(4);
+  });
+
+  test('Test comment counter', async () => {
+    jest.spyOn(dom, 'getComment').mockImplementation(() => {
+      const comments = [
+        {
+          creation_date: '2023-03-12',
+          comment: 'I love this show.',
+          username: 'Wac',
+        },
+        {
+          creation_date: '2023-03-12',
+          comment: 'I love this show.',
+          username: 'Wac',
+        },
+        {
+          creation_date: '2023-03-12',
+          comment: 'I love this show.',
+          username: 'Wac',
+        },
+      ];
+      return comments.length;
+    });
```

Comment:
Please ensure to cover all possible scenarios with your test, especially a case when there is nothing returned from the API (ie 0), this will ensure you write your function to handle such a case and will make your code not throw an unexpected error